package.json

```diff
@@ -1,10 +1,15 @@
 {
   "license": "MIT",
+  "config": {
+    "reporter": "--reporter mocha-multi-reporters --reporter-options configFile=test/reporter-config"
+  },
   "scripts": {
     "start": "node app.js",
     "start:dev": "hof-build watch --env",
```

Comment:
I know you guys are not responsible for this in the package.json file but I require a `.env` file but that's not mentioned in the README.  Also it doesn't say which env variables I need

Reply:
@MaxOSullivan Reckon you could add something about this when you do the readme?

Reply:
add it as tech debt ticket :)